tests/tools/fswalker/fswalker_test.go

```diff
@@ -322,9 +334,21 @@ func TestWalkChecker_filterReportDiffs(t *testing.T) {
 	} {
 		t.Log(tt.name)
 
+		matchers := tt.fields.GlobalFilterMatchers
+
 		chk := &WalkCompare{
-			GlobalFilterMatchers: tt.fields.GlobalFilterMatchers,
+			GlobalFilterFuncs: []func(string, fswalker.ActionData) bool{
+				func(inputStr string, _ fswalker.ActionData) bool {
```

Comment:
What's the difference between this filter and the one above ? (https://github.com/kastenhq/kopia/pull/46/files#diff-7af82b7d8f55353f803ba8a8d1a24f16R192)

Reply:
They're the same; the tests just do a simple `strings.Contains` to flag whether something expects to be filtered or not. This is building the filter function using the input list of filter strings from the test case. Both tests use the same functionality.